cabal-install/tests/UnitTests/Distribution/Solver/Modular/DSL.hs

```diff
@@ -478,7 +478,7 @@ exAvSrcPkg ex =
           -- solver allows unknown extensions/languages when the compiler
           -- supports them.
           let checks = C.checkPackage (srcpkgDescription package) Nothing
-          in filter (not . isUnknownLangExt) checks
+          in filter (\x -> not (isMissingUpperBound x) && not (isUnknownLangExt x)) checks
```

Comment:
Please update the comment above this filter, since the missing bounds check is unrelated to unknown extensions and languages.

Reply:
I updated this comment